### ISSUE TITLE
The forum down error page inconsistently gets affected by `dark-www`

### ISSUE THREAD

**Issue by mybearworld** [ScratchAddons/ScratchAddons#6675]
### Description

When the forums are down, a page is used, which is only sometimes affected by `dark-www`.

### Steps to reproduce

1. Go to the forums when they're down
2. Reload a lot

### Demonstration(s) and additional context

![the forum down image](https://github.com/ScratchAddons/ScratchAddons/assets/130385691/cc3a4dfc-d8a0-4ff3-8662-7210e6101202)
![the forum down image in dark](https://github.com/ScratchAddons/ScratchAddons/assets/130385691/8f5745a9-5fdf-490f-a081-655e31844fb8)

### Environment

Browser: Edge 117
Scratch Addons version: 1.34.1
Operating system: Windows

**Comment by WorldLanguages** [comment]
> Reload a lot

bro :joy:

Relevant lines of code: https://github.com/ScratchAddons/ScratchAddons/blob/9733fdd890854f2023f8a37c947f08c2b6bf5802/background/get-userscripts.js#L269
https://github.com/ScratchAddons/ScratchAddons/blob/d34ee75a4e3684c292d2360739daac13240d502f/content-scripts/cs.js#L52

It's unlikely but this could be caused by Scratch sometimes returning 200 status code with an error page (probably not).
Keep in mind the Scratch Addons status detection is best-effort as there's no reliable way to know the HTTP status code of the current page through JavaScript.

**Comment by mybearworld** [comment]
> > Reload a lot
> 
> bro 😂

What should I have written? That's how you reproduce this

**Comment by mybearworld** [comment]
This'll be a little more difficult to test now that the forums are back up

**Comment by Chiroyce1** [comment]
> This'll be a little more difficult to test now that the forums are back up

I suppose its worth fixing when the forums go down again then lol, this is honestly a non issue since most of the text is stil readable, and the contrast is good enough.

**Comment by mybearworld** [comment]
> I suppose its worth fixing when the forums go down again then lol, this is honestly a non issue since most of the text is stil readable, and the contrast is good enough.

True. It's probably just some race condition thing where some addon expects something from the forums to exist on the page.

**Comment by WorldLanguages** [comment]
The extension tries it best to detect non-2xx error codes, and in those cases it considers the URL to be something like `https://scratch.mit.edu/503/` URLs instead of the one visible in the address bar, and injects userstyles accordingly. This would explain why reloading may result in different styles - the styles directly depend on the pseudo-URL, not the actual URL. See #2416
